products/prisma-cloud/api/cspm/ratelimits.md

```diff
@@ -13,3 +13,12 @@ Rate limits are expressed using two values: `Rate Limit` and `Burst Rate`
 - **Rate Limit**: Number of individual requests per second allowed by the endpoint.
 - **Burst Rate**: Maximum number of concurrent requests allowed in one second.
 
+**User rate limiting**
+
+The rate limiting behavior is based upon the Token Bucket Algorithm and evaluates limits on a per user basis.
+
+Users are identified via a key derived from the authentication token present in all Prisma Cloud API requests.
```

Comment:
@jrdevore This sentence provides context for the note. Also, avoid using foreign words, such as via. Let's address this in the next release.